tests/MongoDB.Driver.Tests/Specifications/crud/CrudTestRunner.cs

```diff
@@ -232,51 +272,23 @@ private void SkipTestIfNeeded(BsonDocument definition, BsonDocument test)
             }
         }
 
-        private class TestCaseFactory : IEnumerable<object[]>
+        private class TestCaseFactory : JsonDrivenTestCaseFactory
```

Comment:
Refactored to use the standard base class which in turn supports JsonDrivenTestCase which integrates better with the Visual Studio test runner.